lib/wallet/http.js

```diff
@@ -478,6 +482,7 @@ class HTTP extends Server {
         subtractIndex: valid.i32('subtractIndex'),
         depth: valid.u32(['confirmations', 'depth']),
         template: valid.bool('template', sign),
+        changeAddress,
```

Comment:
`changeAddress: changeAddress` reads better imo

Reply:
I agree with the above as it keeps it more symmetrical